### ISSUE TITLE
Release RO-Crate 1.2

### ISSUE THREAD

**Issue by elichad** [ResearchObject/ro-crate#353]
Requires all other issues in this milestone to be completed or deferred: https://github.com/ResearchObject/ro-crate/milestone/2

Task list pulled from [RELEASE_PROCEDURE.md](https://github.com/ResearchObject/ro-crate/blob/main/RELEASE_PROCEDURE.md#overview) (may need some updates - please _comment_ with changes so we can make sure the docs are also updated):

The below procedure uses the example of where `1.0` initially is the current release, and `1.1-DRAFT` will become `1.1` as the new current release. Some `docs/` paths below must be updated to `docs/_specification/`

- [x] GitHub housekeeping
  - [x] Create new GitHub milestone for following release - move over issues that won't go into this release
  - [x] All recent fixed issues and pull requests on spec (not other pages) have correct milestone
  - [x] No outstanding open issues on milestone.  
  - [x] No outstanding open pull requests -- or rescheduled for later release (e.g. work in progress).
  - [x] Update team list in `docs/pages/community/community.md` with new members in <https://github.com/ResearchObject/ro-crate/issues/1>
  - [x] Ensure contributors of recent PRs are in RO-Crate team
- [x] Update changelog - note the use of [Semantic Versioning](https://semver.org/) - RO-Crate 1.1 will have GitHub tag `1.1.0`
  - [ ] If this is a later patch update (e.g. typo in `context.jsonld`) then tag as `1.1.1` while all `1.1` strings stays the same. No need for new DOI etc, just add to changelog as new patch version.
- [x] Update author-list in `docs/1.1-DRAFT/index.md` based on team list
- [x] Update `docs/1.1-DRAFT/ro-crate-metadata.json` (base Profile Crate)
- [x] Announce Release Candidate to RO-Crate community - "fire in the hole"!
  - [x] Check for typos, Markdown rendering errors, JSON errors
  - [ ] Test examples and DRAFT @context in <https://json-ld.org/playground/>
  - [x] .. ideally wait 1 week, wait for typo fixes etc.
- [x] Copy `docs/_specification/1.1-DRAFT/` to `docs/_specification/1.1/`
- [x] Search-replace `1.1-DRAFT` to `1.1` in `docs/_specification/1.1/*`
- [x] Prepare a new version of the Zenodo record <https://zenodo.org/record/3406497>
  - [x] **Important** Do not yet click **Publish**!
  - [x] Delete the old uploads
  - [x] Update author list for new members.
  - [x] Update description if needed
  - [x] Update version
  - [x] Update (planned) release date
  - [x] Under _Related identifiers_ set the _Alternative Identifier_ to the future w3id permalink  <https://w3id.org/ro/crate/1.1> and the _Previous version_ to the old permalink <https://w3id.org/ro/crate/1.0>
  - [x] **Important** Copy the pre-allocated DOI from the **draft** Zenodo entry
- [x] Check/update permalink <https://w3id.org/ro/crate/1.1> in `docs/_specification/1.1/*`. Note that it won't work until `docs/_specification/1.1/` folder exists!
- [x] Add **DOI** and (planned) release date to `docs/_specification/1.1/index.md` and `docs/_specification/1.1/ro-crate-metadata.json`
 - [x] Note, there are two DOIs at Zenodo:
   - Version-independent <https://doi.org/10.5281/zenodo.3406497> pointing to latest version
   - Version-specific DOI - which you got from the Zenodo draft 
- [x] Render `docs/_specification/1.1/ro-crate-preview.html` using [ro-crate from npm](https://www.npmjs.com/package/ro-crate)
  - [ ] Make sure output uses the (not-yet released) permalinks <https://w3id.org/ro/crate/1.1> and <https://w3id.org/ro/crate/1.1/context>
- [x] Push `docs/_specification/1.1` to GitHub and verify GitHub Pages rendering and permalinks.
- [x] Convert `docs/_specification/1.1/index.md` to HTML and PDF using `pandoc`
  - `make` can generate PDF (but is not yet updated for new pages -- check output is somewhat sensible)
- [x] Tag in GitHub. [Semantic Versioning](https://semver.org/) so RO-Crate 1.1 will be tagged `1.1.0`
- [x] Prepare release artifacts by copying from `docs/_specification/1.1` to a temporary folder. Rename to include version in filename and upload artifacts to the **release**:
    - `ro-crate-1.1.0.html` 
    - `ro-crate-1.1.0.pdf`
    - `ro-crate-context-1.1.0.jsonld`
    - `ro-crate-metadata.jsonld`
    - `ro-crate-preview.html`
- [x] Update Release notes in GitHub
  - [x] Paste in from Changelog section and the permalink/date headers from `docs/1.1/index.md` 
  - [ ] Example <https://github.com/ResearchObject/ro-crate/releases/tag/1.0.0>
- [x] Upload same artifacts to Zenodo from the GitHub release. 
  - Under _Related identifiers_ modify _Identical to_ to point to the new GitHub release tag
  - Verify and click **Publish**
- [ ] Change search `scope` to use new version in `docs/_config.yml`
- [ ] Announce new version to:
  - [ ] [RO-Crate team](https://github.com/orgs/ResearchObject/teams/ro-crate)
  - [ ] [public-rosc](https://lists.w3.org/Archives/Public/public-rosc/)
  - [ ] [researchobject.org](https://github.com/ResearchObject/ResearchObject.github.io)
  - [ ] Websites of related projects/groups
  - [x] Announcement should only include main point from changelog. Include permalink <https://w3id.org/ro/crate/1.1>  **and** DOI as well as link to RO-Crate community [issue #1](<https://github.com/ResearchObject/ro-crate/issues/1>)
- [x] Prepare next draft e.g. `1.2-DRAFT`:
  - [x] Copy `docs/_specification/1.1-DRAFT/` to `docs/1.2-DRAFT/` and search-replace to `1.2-DRAFT`
  - [ ] Delete `docs/_specification/1.1-DRAFT/*`, but copy released `docs/_specification/1.1/context.jsonld` back to `docs/_specification/1.1-DRAFT/context.jsonld` (avoid breaking existing use)
    - Note: Versions before 1.2 were released on https://www.researchobject.org/ro-crate/1.1/context.jsonld etc. instead of https://www.researchobject.org/ro-crate/specification/1.1/context.jsonld which are maintained in `docs/_compatibility/` as direct copies -- do similar copy-back here, but do not make a new `1.2` or `1.3-DRAFT` folder.

**Comment by elichad** [comment]
Do not interpret this as an announcement that we're releasing 1.2 imminently - this issue is to help prepare for when we do the release in the coming months

**Comment by stain** [comment]
Remaining issues on milestone:
- #300 
- #297 (@dgarijo)

**Comment by stain** [comment]
Updated procedure here (not in `md`) to add milestone housekeeping and checking latest issues are also on the milestone.

**Comment by stain** [comment]
I think for the 1.2 context and profile crate, we don't need to make `docs/_compatibility/1.2` copies, as https://github.com/perma-id/w3id.org/blob/master/ro/crate/.htaccess already has moved to the `specification/` subdirectory.

**Comment by stain** [comment]
Reserved DOI for 1.2.0 is `10.5281/zenodo.13751027`

**Comment by stain** [comment]
#362 should be clarifide before we release.

**Comment by ianhinder** [comment]
This is now unblocked.

**Comment by stain** [comment]
Blocked on #381

**Comment by stain** [comment]
--> Prepare release candidate for 2025-03-13 for vote

**Comment by stain** [comment]
* Release candidate 1.2.0 RC1: https://github.com/ResearchObject/ro-crate/releases/tag/1.2.0-RC1
* Steering commitee vote: https://github.com/ResearchObject/governance/issues/14
* Discussion on RC1: https://github.com/ResearchObject/governance/discussions/13
